store/src/java-test/localconfig-test-copy.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="US-ASCII"?>
```

Comment:
why US-ASCII and not UTF-8?

Reply:
This is just a backup copy of `localconfig-test.xml' https://github.com/Zimbra/zm-mailbox/blob/develop/store/src/java-test/localconfig-test.xml.. We reset it after executing tests